### ISSUE TITLE
luaBeautifier scientific notation gets beautified into a malformed number

### ISSUE THREAD

**Issue by Nexela** [Glavin001/atom-beautify#1588]
# Description

LuaBeautifier seems to put spaces between arithmetic characters.  This breaks scientific notation and creates a malformed number error as well as it makes me a sad panda.

# Input Before Beautification

This is what the code looked like before:

```
Position._epsilon = 1.19e-07
```

# Expected Output

The beautified code should have looked like this:

```
Position._epsilon = 1.19e-07
```

# Actual Output

The beautified code actually looked like this:

```
Position._epsilon = 1.19e - 07
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/cde21a514a13a06eb084f009f45b3e61

# Checklist

I have:
- [ ] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by Glavin001** [comment]
Looking at logs, @Nexela is using version 0.29.18. This is using the new Lua beautifier written for Node.js: https://github.com/Glavin001/atom-beautify/pull/1572

@JoostvDoorn would be the best person to help here.

**Comment by JoostvDoorn** [comment]
Probably @hxsf should look at this.

**Comment by hxsf** [comment]
yes, It looks like I did not expect this problem.

I will fix it.

😃

**Comment by hxsf** [comment]
How about add a option to control whether format operator ?

**Comment by Nexela** [comment]
The option would be nice (I would leave turned on)

Some other things that I would like to see luabeautifer do
command to save with/without beautify (depending on which option is checked)
set the indentation based on the files indentation / indention setting for source

**Comment by hxsf** [comment]
Sorry,  I had a travel 5 days ago, after that, I fixed the bug just now.

**Comment by Glavin001** [comment]
Published to v0.30.0
